django_settings/models.py

```diff
@@ -4,6 +4,9 @@
 from django.contrib.contenttypes.models import ContentType
 from django.contrib.contenttypes import generic
 from django.utils.translation import ugettext_lazy as _
+from django.dispatch import receiver
+from django.db.models.signals import post_syncdb
+from dataapi import initialize_data
```

Comment:
You mean for line 9, don't you? Lines 7 & 8 are absolute imports of django packages.